src/avt/Filters/avtThresholdFilter.C

```diff
@@ -272,7 +367,17 @@ avtThresholdFilter::ProcessOneChunk(avtDataRepresentation *in_dr, bool fromChunk
         in_ds->Register(NULL);
         return in_dr;
     }
-    
+
```

Comment:
You want this if block of code in its original location after the next if block, since this changes the behavior. Specifically, if the user specified they want the output as a point mesh, they would no longer get a point mesh. They would get the mesh from the chunker, which I don't believe will do the conversion.

Reply:
Thanks, I didn't realize that got put in out of order. I've never used the point mesh output. I also fixed an error with vtkm on that path.